admin_handlers.py

```
@@ -0,0 +1,189 @@
+from aiogram import F
```

Ruff (0.12.2)

`aiogram.F` imported but unused

Remove unused import: `aiogram.F`

(F401)